src/client/interface.ts

```diff
@@ -87,6 +92,18 @@ export type TokenNegotiatorEvents =
 	| 'network-change'
 	| 'error'
 
+export interface EventSenderViewLoaded {
```

Comment:
I would like to keep this open to allow flexibility around development. 

- Different views may have different data over time. I don't see the need to lock this in at this early stage and happy we continue. 
- The design to use an object is also allow for flexibility around the data we collect for analytics purposes rather than lock this down at this early stage in development to a single item string

Reply:
@nicktaras I don't consider flexibility a reason not to use types. Types can always be changed over time and it's an easy way to detect issues at build time rather than getting undefined values at runtime that can easily go undetected. Especially since we are sending this data to a database, we shouldn't risk the potential issues.

Reply:
Thanks for the input on this. I am happy to proceed, taking on board your feedback which I'm appreciative of here.
Taking on board the knowledge that should changes come in we haven't locked in a specific interface at this time.